### DISCUSSION TITLE
AdGuard Home

### DISCUSSION THREAD

**Discussion by tv4you2016** [ophub/amlogic-s9xxx-armbian#1756] [Other]
hello @ophub 
What is the possibility of adding AdGuard Home to the list of software?

**Comment by ophub** [comment]
docker?

**Reply by tv4you2016** [reply]
yes. 
Or is there a possibility to install directly on Armbian?

**Comment by ophub** [comment]
This is the docker version I use

**Reply by tv4you2016** [reply]
??

**Comment by LSP0123** [comment]
you can install docker with `sudo armbian-software` and install adguard-home by yourself
